proto/v1alpha1/runtime.proto

```diff
@@ -315,6 +315,15 @@ message RuntimeTeardownRequest {
     string namespace = 2;
     string type = 3;
     string id = 4;
+    Options options = 5;
+}
+
+message Options {
```

Comment:
I feel it might look nicer with `optional`, to signal whether `owner` is set or not. I'm not positive that non-Go protobuf generators will signal the presence of the `message` as `nil`: https://stackoverflow.com/questions/42622015/how-to-define-an-optional-field-in-protobuf-3

Reply:
:cool: I didn't know that `optional` made a comeback.

Reply:
Fixed!